ProcessLib/LIE/Common/FractureProperty.h

```diff
@@ -54,11 +59,28 @@ inline void setFractureProperty(unsigned dim, MeshLib::Element const& e,
     // 1st node is used but using other node is also possible, because
     // a fracture is not curving
     for (int j = 0; j < 3; j++)
-        frac_prop.point_on_fracture[j] = e.getNode(0)->getCoords()[j];
+        frac_prop.point_on_fracture[j] = e.getCenterOfGravity().getCoords()[j];
     computeNormalVector(e, dim, frac_prop.normal_vector);
     frac_prop.R.resize(dim, dim);
     computeRotationMatrix(e, frac_prop.normal_vector, dim, frac_prop.R);
 }
 
+
+inline void setBranchProperty(
+	MeshLib::Node const& branchNode,
+	FractureProperty const& master_frac,
+	FractureProperty const& slave_frac,
+	BranchProperty& branch)
```

Comment:
Consideration: I'd rather return a new Branch object, or maybe create a constructor to the Branch class.

Reply:
done